blog/2023-05-25-tigris-standalone-search/index.mdx

```diff
@@ -0,0 +1,528 @@
+---
+slug: tigris-standalone-search
+title: Enabling awesome search experiences with Tigris Standalone Search
+description: >
+  Enabling awesome search experiences with Tigris Standalone Search
+keywords: [nosql search, standalone search, search engine]
+authors: [leggetter]
+tags: [nosql search, standalone search]
+image: "./tigris-standalone-search.png"
+---
+
+<head>
+  <meta name="twitter:creator" content="@leggetter" />
+</head>
+
+import tigrisConfig from "@site/tigris.config.js";
+
+export const CloudLink = ({ text = "Tigris Cloud" }) => {
+  return <a href={tigrisConfig.signupUrl}>{text}</a>;
+};
+
+Database search is okay, at best. To build awesome search experiences you need a
+dedicated search product.
+
+Earlier this month we
+[released Tigris Search in BETA](/blog/tigris-database-search-sync-beta/) and
+covered how it supports automatic syncronization between Tigris Database and
+Tigris Search, which improves developer productivity through enabling the team
+to focus on shipping features and not setting up and managing database-to-search
+syncronization infrastructure.
+
+But, you many not yet be in the position to migrate all your data over to Tigris
+and you may have a more pressing need to enhance your product with a great
+search experience. Or, you may not store your data in a database; maybe you've
+got all your data in S3, or you've got a statically generated site driven by a
+number of data files stored in your source control.
+
+Whatever the reason, Tigris Standalone Search can help if you're looking for a
```

Comment:
Tigris Standalone Search feature allows you to quickly add search to your existing data.

Reply:
The important point is that it enables developers to quickly add search. This is essentially an iterative way to adopt Tigris, by moving to Tigris workload at a time.

Reply:
Let's not mention `got a statically generated site driven`. As that is not the use case we are targeting.